h2o-extensions/xgboost/src/test/java/hex/tree/xgboost/XGBoostTest.java

```diff
@@ -1261,6 +1242,39 @@ public void testMojoBoosterDump() throws IOException {
     }
   }
 
+  @Test
+  public void testPubDev5816() {
```

Comment:
Minor, yet we should say what it tests, at least in JavaDoc. For convenience, nothing more.